### PR TITLE
Twenty Fifteen Compatibility CSS

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#33501](https://central.tri.be/issues/33501)
_Rel:_ moderntribe/events-pro#138

### CODE REVIEW

File: src/resources/css/tribe-events-full.css [hunk 1]
```diff
@@ -1710,6 +1710,20 @@ input[name*="tribe-bar-"]:-moz-placeholder {
 .tribe-theme-twentyfifteen.events-single.tribe-events-page-template #tribe-events-content {
 	padding: 8.3333% 10%;
 }
+
+.tribe-theme-twentyfifteen.single-tribe_events #tribe-events .tribe_events {
+	margin-left: 0;
+	margin-right: 0;
+	padding-top: 0;
+	box-shadow: none;
```
Comment (borkweb) [inline src/resources/css/tribe-events-full.css]: Alphabetize the properties.

File: src/resources/css/tribe-events-full.css [hunk 2]
```diff
@@ -1710,6 +1710,20 @@ input[name*="tribe-bar-"]:-moz-placeholder {
...
+}
+
+.tribe-theme-twentyfifteen.single-tribe_events #tribe-events .tribe-events-schedule h2 {
+	margin: 0;
+	font-size: 1.2em;
+	display: inline-block;
```
Comment (borkweb) [inline src/resources/css/tribe-events-full.css]: Alphabetize

**Comment by bordoni** [comment]
ping @borkweb

**Comment by borkweb** [comment]
It is so bizarre that the diff isn't visible. I clicked through to look at the file, though, and it looks good.

**Pull request merged**